### PR TITLE
Replace person with team for documentation reviews

### PR DESCRIPTION
**Pull request by osg-grafana**

### CODE REVIEW

**Review by replay — APPROVED** [review]

**Comment by replay** [comment]
Sorry, I approved a bit too quick, there is an error actually because this team is unknown

File: CODEOWNERS [hunk 1]
```diff
@@ -2,4 +2,4 @@
 * @grafana/mimir-maintainers
 
 # Documentation.
-/docs/ @docs-metrics @grafana/mimir-maintainers
+/docs/ @grafana/docs-metrics @grafana/mimir-maintainers
```
Comment (osg-grafana) [inline CODEOWNERS:5]: @replay, I am not sure what is accurate here:

* `@grafana/docs-metrics`
* `@docs-metrics`

Comment (replay) [reply CODEOWNERS:5]: I'm not sure, but I noticed that in the project settings page under "collaborators and teams" I see `docs-squad` but I don't see `docs-metrics`

Comment (osg-grafana) [reply CODEOWNERS:5]: Yes, looks like I do per https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax.

Comment (osg-grafana) [reply CODEOWNERS:5]: > I'm not sure, but I noticed that in the project settings page under "collaborators and teams" I see `docs-squad` but I don't see `docs-metrics`

I dug around and created the new team. :)

Comment (osg-grafana) [reply CODEOWNERS:5]: I then added the team, and added write permissions. The `CODEOWNERS` file is now valid.

**Pull request merged**